draft-smyslov-ipsecme-ikev2-mceliece-00.xml

```diff
@@ -90,17 +93,17 @@
         </section>
 
         <section anchor="mceliece-ikev2" title="Using Classic McEliece in IKEv2">
-          <t> To negotiate the use of the McEliece KEM, an IKEv2 initiator and responder MUST negotiate
+          <t> To negotiate the use of the McEliece KEM, an IKEv2 initiator and responder <bcp14>MUST</bcp14> negotiate

```

Comment:
I wasn't aware of this tag. Does it do any special formatting in the HTML version?